F21/shirishma1/Code Review.py

```diff
@@ -0,0 +1,51 @@
+class LowerCaseConversion():
+   
+    def lowercase(self,text): # if there are no specific conditions this method is used
+        return text.lower()
+        
+    def Irish(self,text):

```

Comment:
My suggestion is you have defined a function for every language it's better to define a function, It reduces the spaces and line of code .

Reply:
Code Readability is good when a function does only one thing and if a new language comes adding another function will not break any other functions.